package.json

```diff
@@ -34,6 +34,8 @@
   "author": "Coveo",
   "license": "Apache-2.0",
   "devDependencies": {
+    "@coveo/semantic-monorepo-tools": "file:.",
```

Comment:
This import can looks funky at first glance, but what it does is it ensure the 'eat-your-own-dog-food' is applied to the letter: `release.mjs` will use the built version of the project to publish itself.